### ISSUE TITLE
New Associate Member Application: ISC2

### ISSUE THREAD

**Issue by hythloda** [ossf/Governance-Committee#18]
This is a new application for an Associate Membership Level.

What is your member company?
**ISC2**

What drives your interest in joining the OpenSSF:
**The ability to collaborate and move forward our mutual interests to secure software**

What are your organization’s aspirations for contributing to the OpenSSF, and how do you anticipate that your membership will bolster the OpenSSF's growth and support
**Currently working on areas of mutual interest around education and global public policy.**

Could you summarize your organization’s contributions to OpenSSF:
**Currently working on areas of mutual interest around education and global public policy.**

How many developers do you expect to have contribute to OpenSSF projects in the next 6-12 months? Are there other roles such as researchers, analysts or any other positions that you plan on contributing:
**Our contributions will be around education and advocacy.**

How do you currently leverage any OpenSSF resources in your organization?
**Education**

Identify which category their organization falls under:
**Non-profit**

Link to your website: 
**[www.isc2.org](http://www.isc2.org/)**

What is your organizational mission statement:
**ISC2 strengthens the influence, diversity and vitality of the field through advocacy, expertise and workforce empowerment that accelerates cyber safety and security in an interconnected world.**

Do you have signing authority for your entire institution? 
**Yes**

**Comment by SecurityCRob** [comment]
I approve of officially bringing ISC2 into our community!  I look forward to our collaboration!

**Comment by hythloda** [comment]
They are now members!
